lib/auth/auth.go

```diff
@@ -1800,6 +1800,12 @@ func (a *Server) NewWebSession(req types.NewWebSessionRequest) (types.WebSession
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	netCfg, err := a.GetClusterNetworkingConfig(context.TODO())
```

Comment:
It was intended to update later because the places that call `NewWebSession` also has other functions called with `context.TODO` example: https://github.com/gravitational/teleport/blob/master/lib/auth/auth.go#L1061

Reply:
I agree with Zac in principle (commented on this in other PRs as well), but context.TODO is sprinkled all over the codebase, with uses similar to this one. A bigger cleanup might be in order, but this goes way beyond this PR.

Personally, I would take either use for this file, but I would probably be more picky for newer code.